### ISSUE TITLE
Extra time setting not being applied to assignment deadlines

### ISSUE THREAD

**Issue by khale** [diderot-edu/diderot-feedback#105]
I'm not sure if I'm using this feature wrong, but I do have some ADA accommodations I need to make for certain students (extra time). For example, I've set some students to have 50% extra time in the "Manage Users -> Extra time column". However this extra time does not appear to have any affect on assignment (submittable chapter) deadlines. I've found that these students end up with the same deadlines as everyone else.

For now I can just manually adjust deadlines for individual students for each assignment, so not a big deal. I also may be misinterpreting this extra time setting.

**Comment by umutacar** [comment]
yes, extra time applies only to exams.  we could try to give the instructor the option of applying them automatically.
